tests/unit/scanners/test.javascript.js

```diff
@@ -318,20 +318,28 @@ describe('JavaScript Scanner', () => {
   // don't change behaviour on us.
   // https://github.com/mozilla/addons-linter/pull/98#issuecomment-158890847
   it('ignores /*global foo*/', () => {
-    const eslint = new ESLint.Linter();
+    const eslint = new ESLint.Linter({ configType: 'eslintrc' });
     const config = { rules: { test: 2 } };
     let ok = false;
 
     eslint.defineRules({
-      test(context) {
-        return {
-          Program() {
-            const foo = getVariable(context.getScope(), 'foo');
-            expect(foo).toBeFalsy();
-
-            ok = true;
-          },
-        };
+      test: {
+        create(context) {
+          const sourceCode = context.sourceCode ?? context.getSourceCode();
+
+          return {
+            Program(node) {
+              const scope = sourceCode.getScope
+                ? sourceCode.getScope(node)
+                : context.getScope();
```

Comment:
Nit: similar comment here on the test side as the previous one related to the backward compatibility logic not being strictly necessary.